doc/source/train/api.rst

```diff
@@ -7,6 +7,73 @@ This page covers advanced configurations for specific frameworks using Train.
 
 For different high level trainers and their usage, take a look at the :ref:`AIR Trainer package reference <air-trainer-ref>`.
 
+.. _train-api-torch-utils:
```

Comment:
I thought these get captured in https://github.com/ray-project/ray/pull/28192/files#diff-c096028dc439275ded6e03c349bd56006c31319501b9bbc46d68994006aa0db2R255-R258 ?

Reply:
Idts- these are not part of the AIR session API

Reply:
then why do we need :no-index:?

Reply:
Updated the PR, PTAL!